src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -492,19 +493,18 @@ std::string ContextAttribute::getLocation(ApiVersion apiVersion) const
 */
 std::string ContextAttribute::renderAsJsonObject
 (
-  ApiVersion          apiVersion,
-  RequestType         request,
-  const std::string&  indent,
-  bool                comma,
-  bool                omitValue
+  ApiVersion   apiVersion,
+  RequestType  request,
+  bool         comma,
+  bool         omitValue
 )
 {
   std::string  out                    = "";
   bool         commaAfterContextValue = metadataVector.size() != 0;
   bool         commaAfterType         = !omitValue || commaAfterContextValue;
 
-  out += startTag(indent, name, false);
-  out += valueTag(indent + "  ", "type",         type,  commaAfterType);
+  out += startTag(name, false);
+  out += valueTag("type",         type,  commaAfterType);
```

Comment:
Old indent no longer necessary

Reply:
You are right, but I prefer not adding new commits to this PR except if really needed (i.e. if you find some criticial bug or problem). The less commits, the easy to rollback if we need so in the future (let's hope no, but better to be conservative). Of course we always can do squash&merge, but I prefer to add unneded complexity. 

Another reason is to be sure that this PR, when merged, mirrors exactly the same changes that the old branch haderding/remove_ngsiv1_indent (yet existing) has.

Reply:
No probs

NTC